docs/introduction.md

```diff
@@ -138,6 +140,43 @@ ncn-m001# sat bash
 (CONTAINER_ID) sat-container# sat status
 ```
 
+## SAT in CSM
+
+In CSM 1.3 and newer, the `sat` command is automatically available on all the Kubernetes NCNs, though it is still
+possible to install SAT as an independent product stream. Installing the SAT product stream results in additional
+supporting components being installed.
+
+- Installing the SAT product stream creates an entry for SAT in the `cray-product-catalog` Kubernetes ConfigMap.
+  Without installing the SAT product stream, there will be no entry for this version of SAT in the output of
+  `sat showrev`.
+
+- The `sat-install-utility` container image is only available with the full SAT product stream. This container image
+  provides uninstall and activate functionality when used with the `prodmgr` command. It is still possible to uninstall
+  previous versions of the SAT product stream.
+
+- The `sat-config-management` git repository in Gitea (VCS) and thus the SAT layer of NCN CFS configuration is
+  only available with the full SAT product stream. See [SAT CFS Configuration](#sat-cfs-configuration-details) below.
+
+- The `docs-sat` RPM package is only available with the full SAT product stream.
+
+### SAT CFS Configuration Details
+
+When the SAT product stream has not been installed, there will be no configuration content for SAT in VCS. Therefore,
+CFS configurations that apply to NCNs (for example, `ncn-personalization`) should not include a SAT layer.
+
+This configuration layer modifies the permissions of files left over from prior installs of SAT, so that the Keycloak
+username used to authenticate to the API gateway cannot be read by users other than `root`. Specifically, it does the
```

Comment:
```suggestion
The SAT configuration layer modifies the permissions of files left over from prior installations of SAT, so that the Keycloak
username that authenticates to the API gateway cannot be read by users other than `root`. Specifically, it does the
```

Reply:
Addressed in 45fae7c23fd959bdca55482f63824e0422d6509c